docs/development.rst

```diff
@@ -254,28 +254,140 @@ HMC inventory file
+
+          contact: <string>             # Optional: Informal reference to a contact for the HMC.
+
+          access_via: <string>          # Optional: Reminder on network setup needed for access.
+
+          ansible_host: <host>          # If real HMC: DNS hostname or IP address of HMC, if
+                                        #   arbitrary string was used as HMC nickname.
+
+          mock_file: <path_name>        # If mocked HMC: Relative path name of HMC mock file.
+          cpcs:                         # CPCs to test against. Can be a subset or all CPCs
+                                        #   managed by the HMC.
+
+            <cpc_name>:                 # CPC name.
+
+              dpm_enabled: <bool>       # Whether the CPC is in DPM mode (true) or classic mode
+                                        #   (false). This is used to include the CPC in pytest
+                                        #   fixtures that select CPCs based on their mode.
+
+              <prop_name>: <prop_value>  # Optional: Additional expected CPC properties, for
+                                        #   use by test functions.
+
+          <var_name>: <var_value>       # Optional: Additional variables for this HMC, for use
+                                        #   by test functions.
+
+      vars:                             # Optional: Additional variables for all HMCs, for use
+                                        #   by test functions.
+        <var_name>: <var_value>
+
+      children:                         # Optional: HMC groups.
+
+        <group_nick>:                   # Nickname of this HMC group.
+
+          hosts:                        # The HMCs in this group.
+
+            <hmc_nick>:                 # Reference to an HMC in this group via its nickname.
+
+              ...                       # Optional: Additional variables to override the ones
+                                        #   inherited from the parent HMC definition. Not
+                                        #   normally needed.
+
+          vars:                         # Optional: Additional variables for the HMCs in this
+                                        #   group.
+            <var_name>: <var_value>
+
+          children:                     # Optional: Grand child groups. Only ever needed when
+            ...                         #   using variable inheritance for some reason.
+                                        #   Can be further nested.
+
+Here is the example HMC inventory file
+`examples/example_hmc_inventory.yaml <https://github.com/zhmcclient/python-zhmcclient/blob/master/examples/example_hmc_inventory.yaml>`_:
+
+.. literalinclude:: ../examples/example_hmc_inventory.yaml
+   :language: yaml
+
+In that example HMC inventory file, the following nicknames of single HMCs are
+defined:
+
+* HMC1 - The HMC at 10.11.12.13.
+* MOCKED_Z13_CLASSIC - The mocked HMC defined in mock file
```

Comment:
No, we actually have two mock files for Z13, that are referenced in the example HMC inventory file, so this is a real example. If we want to move that to a newer generation, it would need to be done consistently, including these mock files. I have created https://github.com/zhmcclient/python-zhmcclient/issues/1133 for that.